src/Components/Common/PrescriptionBuilder.gen.tsx

```diff
@@ -7,7 +7,7 @@ import * as React from 'react';
 // tslint:disable-next-line:no-var-requires
 const PrescriptionBuilderBS = require('./PrescriptionBuilder.bs');
 
-import {t as Prescription__Prescription_t} from '@coronasafe/prescription-builder/src/Types/Prescription__Prescription.gen';
+import {t as Prescription__Prescription_t} from './Prescription__Prescription.gen';
```

Comment:
Why did you change this 🤔

Reply:
I didn't, it automatically happened. I thought it was part of some scripts.

Reply:
so should i change it back?

Reply:
Yes please

Reply:
rolled the changes back. see commit 2d7304c